packages/components/src/pages/VsCodeExtension.vue

```diff
@@ -173,14 +173,19 @@
             </button>
           </v-popper>
           <v-popper
-            v-if="showDownload && !isGiantAppMap"
+            v-if="showDownload"
             class="hover-text-popper"
             text="Export in SVG format"
```

Comment:
Ability to download the AppMap from the AppMap itself would be pretty useful for things like the Confluence plugin - so the user who's viewing the page can download the data directly.

As I've thought about this, it would also be useful to include the filter settings in the AppMap itself, say under a `filter` key at the top level. This would be a way that we could make the filter settings portable with the AppMap. Suppose I am looking at an AppMap in my code editor and I've configured the filter settings. Then I click the "Export AppMap data", it opens the AppMap file in my system Finder / Explorer with the filter settings embedded. I drop this into Confluence and my filter settings will be preserved.